Algorithm.Python/FractionalQuantityRegressionAlgorithm.py

```diff
@@ -47,7 +47,7 @@ def Initialize(self):
 
         self.SetTimeZone(DateTimeZone.Utc)
 
-        security = self.AddSecurity(SecurityType.Crypto, "BTCUSD", Resolution.Daily, Market.GDAX, False, 3.3, True)
+        security = self.AddSecurity(SecurityType.Crypto, "BTCUSD", Resolution.Daily, Market.GDAX, False, d.Decimal(3.3), True)
```

Comment:
I'm unfamiliar with `d.Decimal` -- am I correct to assume that it's a function we added to pythonnet to convert float->decimal? Do we need documentation updates so users know to use this?

Reply:
Yes, you are correct. I will scan the docs and see if we have examples without it and update them accordingly.